### PR TITLE
Migrate Java to Kotlin

### PR DESCRIPTION
**Pull request by cuong0993**
Android development moved to Kotlin. Please consider merging.

### CODE REVIEW

File: godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.kt [hunk 1]
```diff
@@ -0,0 +1,301 @@
+/*************************************************************************/
+/*  GodotGooglePlayBilling.java                                                    */
```
Comment (timoschwarzer) [inline godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.kt:2]: .kt :)

File: godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/utils/GooglePlayBillingUtils.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
+/*************************************************************************/
+/*  GooglePlayBillingUtils.java                                                    */
```
Comment (timoschwarzer) [inline godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/utils/GooglePlayBillingUtils.kt:2]: .kt

**Review by timoschwarzer — CHANGES_REQUESTED** [review]

**Comment by timoschwarzer** [comment]
Sounds good to me, thank you!

File: godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/utils/GooglePlayBillingUtils.kt (hunk 2 above)
Comment (cuong0993) [reply godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/utils/GooglePlayBillingUtils.kt]: Fixed 👍

File: godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.kt (hunk 1 above)
Comment (cuong0993) [reply godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.kt]: Fixed 👍

**Pull request closed without merging**